components/EditorialStory/EditorialStory.tsx

```diff
@@ -0,0 +1,45 @@
+import { IPost } from "components/Post/Post";
+import PostPreview from "components/PostPreview/PostPreview";
+
+interface IEditoralStory {
+  posts: Array<IPost>;
+}
```

Comment:
1️⃣ Leave the interface in the index.ts file. Move the component to its own file. E.g. component.tsx

Take a look at the README where I explain who to organize your component code: https://github.com/MZegarraGlobant/blog-onboarding-exercise-Mike#how-to-create-components